bullet_train-fields/app/javascript/controllers/dependent_fields_frame_controller.js

```diff
@@ -8,7 +8,7 @@ export default class extends Controller {
   }
   static classes = [ "loading" ]
   
-  updateFrameFromDependentField(event) {
+  updateFrameFromDependableField(event) {
```

Comment:
Am I right in assuming that `DependableField` represents the main field being triggered which then updates the `DependentField`? Just want to make sure I'm getting these naming conventions right.

Reply:
You got it 👍